bee-storage/bee-storage-rocksdb/Cargo.toml

```diff
@@ -21,7 +21,9 @@ async-trait = "0.1"
 futures = "0.3"
 num_cpus = "1.13"
 pin-project = "1.0"
-rocksdb = { version = "0.16", default-features = false }
+rocksdb = { git = "https://github.com/rust-rocksdb/rust-rocksdb", branch = "master", default-features = false }
```

Comment:
can't we use a specific commit just in case?

Reply:
Sure